modules/cluster-tagging.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * machine_management/adding-rhel-compute.adoc
+// * machine_management/more-rhel-compute.adoc
+
+
+[id="cluster-tagging_{context}"]
+= Tagging an existing cluster as owned
+
+A cluster uses the value of the `kubernetes.io/cluster/<clusterid>,Value=(owned|shared)` tag to determine which resources belong to the AWS cluster. These resources include:
+
+* All hosts.
+* All relevant load balancers to be used in the AWS instances.
+* All relevant security groups to be used with the AWS instances.
+* All EBS volumes.
++
+[TIP]
+====
+Run the following command to find all EBS volumes that must be tagged:
+[source,terminal]
+----
+$ oc get pv -o json|jq '.items[].spec.awsElasticBlockStore.volumeID'
+----
+====
+
+.Procedure
+
+. With {op-system-base} compute machines, set the cluster to owned with the `kubernetes.io/cluster/<clusterid>=owned` tag using the appropriate cluster ID.
+. Restart the master services.
+. Restart the node service on all available nodes.
```

Comment:
do we need to restart master services and node service here?

Reply:
Based on the original documentation, it did require that users restart both master services then node services.

Reply:
restarting is not needed